analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/EnvironmentVariableScope.cs

```diff
@@ -34,6 +34,11 @@ public bool EnableConcurrentAnalysis
         {
             set => SetVariable(SonarDiagnosticAnalyzer.EnableConcurrentExecutionVariable, value.ToString());
         }
+
+        public bool EnableRazorAnalysis
+        {
+            set => SetVariable(SonarAnalyzer.Extensions.SyntaxTreeExtensions.EnableRazorAnalysisVariable, value.ToString());
```

Comment:
I find it strange that we don't test non bool variables (as users may introduce typos and we should not fail)

Reply:
Do you think it is worth investing the effort in a test that will be removed in 2 weeks together with the environment variable?  As you mentioned the variable is internal and only users who wander around our codebase will know it exists.
The main reason this variable exists is the fact that we wanted to merge to master because working on a feature branch with all the current limitations is very very very painful.

Reply:
probably not, indeed

The previous one is public though, but it's outside of the scope of this PR